src/main/java/bio/terra/service/snapshotbuilder/utils/QueryBuilderFactory.java

```diff
@@ -5,11 +5,15 @@
 import org.springframework.stereotype.Component;
 
 @Component
-public class CriteriaQueryBuilderFactory {
```

Comment:
If we're making a general QueryBuilderFactory, should we put `SearchConceptsQueryBuilder` in it as well?

Reply:
I think that would make sense but doesn't really fit in with this PR. I created https://broadworkbench.atlassian.net/browse/DC-952 for this work.